platform/openide.loaders/src/org/openide/loaders/DataNode.java

```diff
@@ -540,6 +540,9 @@ protected Sheet createSheet () {
                 ss.put(new ExtensionProperty());
             }
             ss.put(new SizeProperty());
+            ss.put(new MIMETypeProperty());
```

Comment:
Deserves a note in `apichanges.xml` of `openide.loaders`.

Wouldn't it be (from a usability standpoint) better to have just a single property (mime type) with custom property editor to show the details? Three properties for MIME type, it's display name and a filter pattern seem a bit too overkill for majority of users.